plugins/cloudtrail/pkg/cloudtrail/config.go

```diff
@@ -18,9 +18,9 @@ package cloudtrail
 
 // Struct for plugin init config
 type PluginConfig struct {
-	S3DownloadConcurrency int  `json:"s3DownloadConcurrency" jsonschema:"description=Controls the number of background goroutines used to download S3 files (Default: 1)"`
-	SQSDelete             bool `json:"sqsDelete" jsonschema:"description=If true then the plugin will delete sqs messages from the queue immediately after receiving them (Default: true)"`
-	UseAsync              bool `json:"useAsync" jsonschema:"description=If true then async extraction optimization is enabled (Default: true)"`
+	S3DownloadConcurrency int  `json:"s3DownloadConcurrency" jsonschema:"title=S3 download concurrency,description=Controls the number of background goroutines used to download S3 files (Default: 1),default=1"`
```

Comment:
Should we remove the default from the description?
Not totally sure. @jasondellaluce wdyt?

Reply:
I think we may want to leave it for now, mostly because it is used as-is to print out field information in some places (e.g. `falco --list`). We can remove this once we have a more coherent way to obtain the default value from the schema in the tools using it.